kytos/utils/users.py

```diff
@@ -0,0 +1,144 @@
+
+    def __init__(self, controller=None):
+        """If controller is not informed, the necessary paths must be.
+
+        If ``controller`` is available, NApps will be (un)loaded at runtime and
+        you don't need to inform the paths. Otherwise, you should inform the
+        required paths for the methods called.
+
+        Args:
+            controller (kytos.Controller): Controller to (un)load NApps.
+            install_path (str): Folder where NApps should be installed. If
+                None, use the controller's configuration.
+            enabled_path (str): Folder where enabled NApps are stored. If None,
+                use the controller's configuration.
+        """
+        self._controller = controller
+        self._config = KytosConfig().config
+        self._users_client = UsersClient()
+
+    def register(self):
+        """Method used to register a new user.
+
+        This method will as for user attributes and create that in
+        Napps server, when All required fields is filled.
+
+        Returns:
+            result(string): Response of user registration process.
+        """
+        user = {}
+
+        print('--------------------------------------------------------------')
+        print('Welcome to the user registration process.')
+        print('--------------------------------------------------------------')
+        print("To continue you must fill the follow fields.")
+        for attribute, value in self.attributes.items():
+
+            is_required = attribute in self.required
+            question = value['question']
+            pattern = value['pattern']
+
+            if attribute != 'password':
+                user[attribute] = self.make_question(question, pattern,
+                                                     is_required)
+            else:
+                user[attribute] = self.make_question(question, pattern,
+                                                     password=True)
+
+        return self._users_client.register(user)
+
+    def make_question(self, question, pattern=NAME_PATTERN, required=False,
+                      password=False):
+        """Method used to make a question and get the input values.
+
+        This method will validade the input values.
+        Args:
+            question(string): Question used to ask for input value.
+            pattern(string): Pattern to validate the input value.
+            required(bool): Boolean value if the input value is required.
+            password(bool): Boolean value to get input password with mask.
+        Returns:
+            input_value(string): Input value validated.
+        """
+        input_value = ""
+
+        while not input_value:
+            input_value = getpass(question) if password else input(question)
+
+            if not input_value:
+                if not required:
+                    break
+                else:
+                    continue
```

Comment:
Change this block [L115-L119] by:
```python
if not (input_value or required):
    break
```
(I Think that this would work, please, confirm!)